modular_skyrat/modules/departmentization/cargo_technician.dm

```diff
@@ -0,0 +1,12 @@
+/datum/outfit/job/cargo_tech
+	name = "Cargo Technician"
+	jobtype = /datum/job/cargo_technician
+
+	backpack_contents = list(
+		/obj/item/boxcutter = 1,
+	)
+	id_trim = /datum/id_trim/job/cargo_technician
+	uniform = /obj/item/clothing/under/rank/cargo/tech/skyrat/turtleneck
```

Comment:
Why are warehouse workers being given turtlenecks as their standard work wear?

Reply:
They push around crates and deliver mail, they're called Warehouse Technicians. Unless you intend to remove that part of their job then they're still Amazon workers. Just with a fancy name.

You could have just said 'this is the only one with a skirt variant'.

Reply:
the alternative was a mil-larp get up(gorka); i was telling you, that per-their-policy, and this change, they won't be 'amazon workers,' (even if they haven't been for awhile); and they won't likely go back to being in that ballpark either, so conforming stuff to fit a design that isn't theirs (as said above), doesn't line up.

Reply:
Policy could say that Cargo has to have five stripper poles, only speak in limerick and have to obey anyone with an even amount of vowls in their name.

It will not change the fact that *mechanically* they push crates. They are amazon workers.

But yes, A turtleneck, even if overtly formal for a underpaid crate pusher is better than _**more**_ military LARP.

Reply:
mechanics have no precedence over decisions like this, so far you're at a double negative